charts/brigade/requirements.lock

```diff
@@ -9,4 +9,4 @@ dependencies:
   repository: https://brigadecore.github.io/charts
   version: 0.2.0
 digest: sha256:b556135c790a819bc1c2914dde6aec89c2b7467298b36000229229aaf9800c09
-generated: "2019-08-16T19:12:40.678501-04:00"
+generated: "2019-11-21T12:27:31.572531-07:00"
```

Comment:
I'm having a lapse in memory right now. Why would the `requirements.lock` file have changed in this PR? Don't we need to release each of the individual "leaf" charts, then rev the requirements in the brigade chart's `requirements.yaml` and only then does the brigade chart's `requirements.lock` change? Or am I mis-remembering the process?

Reply:
Thank you @krancour - this must have slipped through when I was testing with the requirement overrides to pull sub-charts from my local filesystem.  As you said, this file should not have been changed in this PR, but will be in the subsequent sub-chart version bump PR.  Removed.